fml/message_loop_task_queues.cc

```diff
@@ -318,13 +340,13 @@ bool MessageLoopTaskQueues::HasPendingTasksUnlocked(
     return true;
   }
 
-  const TaskQueueId subsumed = entry->owner_of;
-  if (subsumed == _kUnmerged) {
-    // this is not an owner and queue is empty.
-    return false;
-  } else {
-    return !queue_entries_.at(subsumed)->task_source->IsEmpty();
+  auto& subsumed_set = entry->owner_of;
+  for (auto& subsumed : subsumed_set) {
+    if (!queue_entries_.at(subsumed)->task_source->IsEmpty()) {
+      return true;
+    }
```

Comment:
I'm not familiar with this method, @iskakaushik can you double check this is correct?

Reply:
this looks correct to me